examples/demo/showcase/main.py

```diff
@@ -75,8 +75,8 @@ def go_screen(self, idx):
         self.root.ids.sm.switch_to(self.load_screen(idx), direction='left')
         self.update_sourcecode()
 
-    def go_higherarchy_previous(self):
-        ahr = self.higherarchy
+    def go_heirarchy_previous(self):
+        ahr = self.heirarchy
```

Comment:
This should be ready to merge now.
1.  Fixed the hierarchy misspellings.
2.  Fixed the extra line.
3.  Fixed the long-line style warnings from 'make style'.

I solemnly swear to make the next set of commits on a branch other than
master. :).    Next set of commits will be either some tests or fixing the
html doc generation.
https://groups.google.com/forum/#!topic/kivy-dev/4_DzJUuZAXM

Charles



On Mon, Jan 26, 2015 at 12:45 PM, Charles Merriam <charles.merriam@gmail.com
> wrote:

> My spelling fails as well!  :)
>
>
> On Mon, Jan 26, 2015 at 12:16 PM, dessant <notifications@github.com>
> wrote:
>
>> In examples/demo/showcase/main.py
>> <https://github.com/kivy/kivy/pull/2872#discussion_r23560685>:
>>
>> > @@ -75,8 +75,8 @@ def go_screen(self, idx):
>> >          self.root.ids.sm.switch_to(self.load_screen(idx), direction='left')
>> >          self.update_sourcecode()
>> >
>> > -    def go_higherarchy_previous(self):
>> > -        ahr = self.higherarchy
>> > +    def go_heirarchy_previous(self):
>> > +        ahr = self.heirarchy
>>
>> I mean the correct form is "hierarchy", we could merge it after the fix.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/kivy/kivy/pull/2872/files#r23560685>.
>>
>
>